### PR TITLE
 Adds support for running Xenon on a computer, and using Plethora's chest.drop instead

### PR DESCRIPTION
**Pull request by ghost**
Use "Squash and Merge" please

### CODE REVIEW

File: src/sections/inventory.lua [hunk 1]
```diff
@@ -109,15 +109,19 @@ local function dispense(mcname, count)
 
     for i = #slotList[mcname], 1, -1 do
       local chestPeriph = chestPeriphs[slotList[mcname][i][3]]
-      local amountPushed = chestPeriph.pushItems(config.self, slotList[mcname][i][1], count)
+      if not (config.outChest and targetChest == config.outChest) then
+        local amountPushed = chestPeriph.pushItems(config.self, slotList[mcname][i][1], count)
```
Comment (incinirate) [inline src/sections/inventory.lua]: Er, you have to define `amountPushed` outside the `if` because scoping

**Review by incinirate — CHANGES_REQUESTED** [review]
Change that and then it looks good

**Comment by incinirate** [comment]
lgtm

**Comment by ghost** [comment]
Thanks

**Pull request merged**